### PR TITLE
ci: Enable feature_asmap.py test on native Windows

### PR DESCRIPTION
**Pull request by hebasto**
This PR is a [follow up](https://github.com/bitcoin/bitcoin/pull/23084#issuecomment-927077970) of #23084. It enables the `feature_asmap.py` functional test which is fixed in #23084.

### CODE REVIEW

**Comment by fanquake** [comment]
>  It enables the feature_asmap.py functional test which is fixed now.

Fixed when, with what change? Probably easier to just mention this in #23102.

**Comment by hebasto** [comment]
> Fixed when, with what change?

In #23084.

> Probably easier to just mention this in #23102.

Agree. @MarcoFalke ?

**Comment by hebasto** [comment]
Rebased on top of the merged #23102.

> Probably easier to just mention this in #23102.

Is no longer relevant.

File: .cirrus.yml [hunk 1]
```diff
@@ -152,7 +151,7 @@ task:
     - netsh int ipv4 set dynamicport tcp start=1025 num=64511
     - netsh int ipv6 set dynamicport tcp start=1025 num=64511
     # TODO enable '--extended' and drop '--exclude'.
```
Comment (MarcoFalke) [inline .cirrus.yml:153]: :eyes:

Comment (MarcoFalke) [reply .cirrus.yml:153]: Also, wondering if `--extended` pass by now

Comment (hebasto) [reply .cirrus.yml]: > Also, wondering if `--extended` pass by now

[Timed out](https://cirrus-ci.com/task/5949285465325568).

**Comment by hebasto** [comment]
Updated comment.

**Comment by MarcoFalke** [comment]
cr ACK 4befc8f1c0a8ecb4293eedcf8292ac9ccb86011b

**Pull request merged**